src/controllers/timeZoneAPIController.js

```diff
@@ -0,0 +1,23 @@
+const timeZoneAPIController = function () {
+    const getTimeZoneAPIKey = (req,res) => {
+        const requestorRole = req.body.requestor.role;
+        const PremiumRoles = ['Manager', 'Administrator', 'Core Team'];
+        const isPremiumRequestor = !!((PremiumRoles.includes(requestorRole)));
+        const premiumKey = process.env.TIMEZONE_PREMIUM_KEY;
+        const commonKey = process.env.TIMEZONE_COMMON_KEY;
```

Comment:
I need to configure deployment service to add these two variables. Currently I will be adding same key for both of these use cases, but this feature can be used on each user profile in future in which case we can have premium key access to selected user classes and add the premium key to the deployment configurations when we are ready for it. 
@chrisweilacker I might need your help to add those configurations and also please review this PR once you get a chance, thanks!